core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -208,8 +214,11 @@ <h3>Advanced Features</h3>
                 </span>
               </button>
             </span>
-            <span class="col-lg-8 col-md-8 col-sm-8 help-block" style="font-size: smaller;">
-              This allows the user to categorise answer groups as correct or incorrect.
+            <span ng-if="!isCorrectnessFeedbackEnabled()" class="col-lg-8 col-md-8 col-sm-8 help-block" style="font-size: smaller;">
```

Comment:
@ishucr7 Can you send two screenshots of this (showing the two states) to Joe and get feedback?

Reply:
Sure!